testing/runner.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to create new test projects for the Google Cloud Run team
+set -e
+
+export TEAM_FOLDER=859004177471
+export IAM_OWNER=group:cloud-devrel@twosync.google.com
+
+./test-project-setup.sh -v
```

Comment:
Let's make this more generalized:

```suggestion
exec "$@"

```

Then in our own docs/usage/etc, we would use `runner.sh test-project-setup.sh`, that way if we have other scripts we have a common way to set the environment.